source/agora/test/PeriodicCatchup.d

```diff
@@ -94,7 +94,7 @@ private class NodeManager (): TestAPIManager
         if (this.nodes.length < 1)  // Use first node as test node
         {
             assert(conf.validator.enabled);
-            log.trace("Create node {} as cathup test node", this.nodes.length);
+            log.trace("Create node #{} ({}) as catchup test node", this.nodes.length, conf.validator.key_pair.address);
```

Comment:
How about adding a line break because the line gets too long.

Reply:
Updated